### PR TITLE
Documentation for FritzBox call monitor

### PR DESCRIPTION
**Pull request by DavidMStraub**
This is the documentation for the component suggested in [this PR](https://github.com/home-assistant/home-assistant/pull/2791).

### CODE REVIEW

File: source/_components/sensor.fritzbox_callmonitor.markdown [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: "FRITZ!Box"
+description: "Instructions how to integrate a phone call monitor for AVM FRITZ!Box routers into Home Assistant."
+date: 2016-08-13 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: avm.png
+ha_category: Sensor
+ha_release: "0.25.2"
```
Comment (balloob) [inline source/_components/sensor.fritzbox_callmonitor.markdown]: Please change this to 0.27

File: source/_components/sensor.fritzbox_callmonitor.markdown [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: "FRITZ!Box"
+description: "Instructions how to integrate a phone call monitor for AVM FRITZ!Box routers into Home Assistant."
+date: 2016-08-13 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: avm.png
```
Comment (balloob) [inline source/_components/sensor.fritzbox_callmonitor.markdown]: The logo is not included ?

Comment (balloob) [inline source/_components/sensor.fritzbox_callmonitor.markdown]: nevermind, it's already there :-)

File: source/_components/sensor.fritzbox_callmonitor.markdown [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ha_category: Sensor
```
Comment (balloob) [inline source/_components/sensor.fritzbox_callmonitor.markdown]: Can you change this to `System Monitor`

I actually don't like that category either but better then Sensor (it used to be the default but I've been cleaning it up)

**Comment by DavidMStraub** [comment]
Sure, done.

**Comment by fabaff** [comment]
Thanks

**Pull request merged**